Re-written Cumulocity Platform Introduction (#2466)
SoftwareAG/c8y-docs: content/concepts/concepts-introduction.md

---
weight: 10
title: Introduction to the Cumulocity IoT platform
layout: bundle
aliases:
  - /concepts/introduction/
sector:
  - getting_started
---

> {{< product-c8y-iot >}} is a robust, secure and scalable Internet of Things (IoT) platform designed to empower business to:

Reviewer: This construction should be followed by:
- Integrate devices, manage devices, and so on
But I think I would better rephrase this sentence so that we can keep Device integration, Device management and so on and use these terms as links to the respective sections
Perhaps: 
"platform designed to empower business in the following aspects:

> 1. **Device Integration**: Connect any type of device, from simple sensors to complex machinery, ensuring reliable and secure data transmission.

Reviewer: ```suggestion
1. **[Device integration](/concepts/concepts-introduction/#device-integration)**: Connect any type of device, from simple sensors to complex machinery, ensuring reliable and secure data transmission.
```

> 2. **Device Management**: Monitor, control, and manage all remotely connected equipment throughout its entire lifecycle, from deployment to retirement.

Reviewer: ```suggestion
2. **[Device management](/concepts/concepts-introduction/#device-management)**: Monitor, control, and manage all remotely connected equipment throughout its entire lifecycle, from deployment to retirement.
```

> 3. **IoT Data Management**: Prepare, normalize, and enrich real-time and historical data, creating a unified view of your equipment.

Reviewer: ```suggestion
3. **[IoT data management](/concepts/concepts-introduction/#iot-data-management)**: Prepare, normalize, and enrich real-time and historical data, creating a unified view of your equipment.
```

> 4. **Application Enablement**: Visualize and analyze IoT data, integrate it with existing business systems, and rapidly build custom IoT applications and solutions to meet your business needs.

Reviewer: ```suggestion
4. **[Application enablement](/concepts/concepts-introduction/#application-enablement)**: Visualize and analyze IoT data, integrate it with existing business systems, and rapidly build custom IoT applications and solutions to meet your business needs.
```

> 5. **Analytics**: Automate, optimize, and remotely control equipment operations using intelligent analytics and real-time decision-making.

Reviewer: ```suggestion
5. **[Analytics](/concepts/concepts-introduction/#analytics)**: Automate, optimize, and remotely control equipment operations using intelligent analytics and real-time decision-making.
```

> 6. **Platform Management**: Manage configurations, security settings, and tenant structures in the cloud, or deploy the platform on your premises for full control. 

Reviewer: ```suggestion
6. **[Platform management](/concepts/concepts-introduction/#platform-management)**: Manage configurations, security settings, and tenant structures in the cloud, or deploy the platform on your premises for full control. 
```

![Platform overview](/images/concepts-guide/platform_overview.png)

> This guide introduces the core concepts and capabilities of {{< product-c8y-iot >}}, catering to IoT architects, automation engineers, industrial engineers, and developers.

Reviewer: ```suggestion
This section introduces the core concepts and capabilities of {{< product-c8y-iot >}}, catering to IoT architects, automation engineers, industrial engineers, and developers.
```

  - getting_started
---

{{< product-c8y-iot >}} is a robust, secure and scalable Internet of Things (IoT) platform designed to empower business to:

1. Connect, monitor, and manage any remotely connected equipment
    
2. Prepare, normalize, organize and enrich the real-time and master data from the connected equipment
    
3. Visualise, analyse and integrate IoT data with existing business systems
    
4. Rapidly build and deploy tailored IoT applications and solutions
    
5. Control, automate and optimise equipment operations remotely
    

> ![Platform Overview](/images/concepts-guide/platform_overview.png)

Reviewer 2: ```suggestion
![Platform overview](/images/concepts-guide/platform_overview.png)
```

> This guide introduces the core concepts and capabilities of Cumulocity IoT, catering to IoT Architects, Automation Engineers, Industrial Engineers, and Developers.

Reviewer 2: ```suggestion
This guide introduces the core concepts and capabilities of {{< product-c8y-iot >}}, catering to IoT architects, automation engineers, industrial engineers, and developers.
```
